### PR TITLE
Add modules & plugins overview pages - Partially closes #1417

### LINKED ISSUE #1417: Add new/missing content to general docs
### Description

Add missing content:

- overview pages
- api page

### PR DESCRIPTION
**Pull request by Tschakki**
Partially closes #1417

### CODE REVIEW

File: modules/ROOT/pages/plugins/index.adoc [hunk 1]
```diff
@@ -1 +1,42 @@
...
+|Standalone or included in the Lisk SDK?
+|Description
+
+| xref:{url_plugin_dashboard}[]
+|standalone
+|Provides a user interface which enables developers to conveniently view and communicate with the blockchain application during the development.
+
+| xref:{url_plugin_faucet}[]
+|standalone
+|Provides a user interface to receive an certain amount of tokens on request.
```
Comment (chris529) [inline modules/ROOT/pages/plugins/index.adoc]:
```suggestion
|Provides a user interface to receive a certain amount of tokens on request.
```

File: modules/ROOT/pages/plugins/index.adoc [hunk 2]
```diff
@@ -1 +1,42 @@
...
+
+| xref:{url_plugin_forger}[]
+|included
+|Monitors configured delegates forging activity, and voters information.
+
+| xref:{url_plugin_httpapi}[]
+|included
+|Provides basic HTTP API endpoints to get running node information.
```
Comment (chris529) [inline modules/ROOT/pages/plugins/index.adoc]:
```suggestion
|Provides basic HTTP API endpoints to get the running node information.
```

**Review by chris529 — COMMENTED** [review]
Looks good, just a couple of very minor points 👍

File: modules/ROOT/pages/plugins/index.adoc [hunk 3]
```diff
@@ -1 +1,42 @@
...
+|Name
+|Standalone or included in the Lisk SDK?
+|Description
+
+| xref:{url_plugin_dashboard}[]
+|standalone
+|Provides a user interface which enables developers to conveniently view and communicate with the blockchain application during the development.
+
+| xref:{url_plugin_faucet}[]
+|standalone
```
Comment (shuse2) [inline modules/ROOT/pages/plugins/index.adoc]: im not sure if `standalone` is appropriate here

File: modules/ROOT/pages/modules/index.adoc [hunk 4]
```diff
@@ -1 +1,44 @@
...
+
+It is responsible for verifying signatures of a transaction from multi-signature and non-multi-signature accounts.
+It also handles the registration of multi-signature accounts.
+
+| xref:{url_module_sequence}[]
+|Handles all logic related to the nonce.
+
+It should verify the nonce for all transactions and increment it if valid.
+
+| xref:{url_module_token}[]
```
Comment (shuse2) [inline modules/ROOT/pages/modules/index.adoc]: I think it's better to order them by `moduleID` same for navigation

**Review by shuse2 — APPROVED** [review]

**Comment by Tschakki** [comment]
Was merged during GitHub hickup, so I close it manually, same as #1423

**Pull request closed without merging**